8.x/theme-development/directories.md

```diff
@@ -1,3 +1,131 @@
+**19**| `is_location_limited`        | boolean      | Specifies the restriction of sales of some products for the provinces.
+**20**| `description`                | string       | Specifies the product directory description.
+**21**| `cmc_id`                     | integer      | CMC (Customer Meta Category), checks whether information should be taken from the customer when buying the product or not purchasing the product.
+**22**| `force_show_landing`         | boolean      | Specifies whether this directory is displayed on the landing page by default or non-landing.
+**23**| `inaccessibility_type`       | integer      | Shows the different ways you deal with the non-existence of this directory.
+**24**| `notice`                     | string       | Specifies a notification for each product in the directory.
+
+**NOTE:** Example of URL part and URL full:
+
+`directory` is the `url_part` for the directory.
+```
+/path/to/the/specific/directory
+```
+
+`/path/to/the/specific/directory` is the `url_full` for the directory.
+```
+/path/to/the/specific/directory
+```
+
+**NOTE:** `content_type` stores the contents of the directory, which are three types. You can see them below:
+
+```php
+class DirectoryType extends BaseEnum
+{
+    const REAL=1;
+    const BLOG=2;
+    const PRODUCT=3;
+}
+```
+
+### Relationship directory table
+
+A `directorie` has some relationships with other entity types existing in the system as shown below:
+
+Num   | Property              | Datatype              | Description
+------|-----------------------|-----------------------|-----------
+**1** | `parentDirectory`     | Directory             | Shows the parent directory.
+**2** | `directories`         | Directory[]           | Specifies hierarchically all the parents of a directory.
+**3** | `leafProducts`        | Product[]             | Specifies products node that has a link from its parent node.
+**4** | `products`            | Product[]             | Shows products connected to this directory.
+**5** | `leafArticles`        | Article[]             | Specifies articles node that has a link from its parent node.
+**6** | `articles`            | Article[]             | Shows articles connected to this directory.
+**7** | `tags`                | Tag[]                 | Specifies tags.
+**8** | `badges`              | Badge[]               | Specifies badges.
+**9** | `customerMetaCategory`| CustomerMetaCategory  | CMC (Customer Meta Category).
+**10**| `discountGroup`       | DiscountGroup         | Specifies a price reduction plan for a directory products.
+
+### Show directory properties
+
+`$directory` and `$web_page` variable is passed from the controller to the blade, and the content of the directory can be displayed using this variables.
+
+**NOTE:** The SEO of all you web pages is managed in directories, note that SEO information is stored in the `$web_page` object.
```

Comment:
**NOTE:** If you are considering to show the SEO contents in the web pages, It's important to know that `seo_keywords` and `seo_description` of each web page is stored in the `$web_page` variable. So that if you want to get desired data use `$directory->webPage` or `$web_page` directly.